plugins/PostProcessingPlugin/scripts/DisplayRemainingTimeOnLCD.py

```diff
@@ -0,0 +1,86 @@
+
+from ..Script import Script
+
+import re
+import datetime
+
+
+class DisplayRemainingTimeOnLCD(Script):
+
+    def __init__(self):
+        super().__init__()
+
+
+    def getSettingDataString(self):
+        return """{
+            "name":"Disaplay Remaining Time on LCD",
+            "key":"DisplayRemainingTimeOnLCD",
+            "metadata": {},
+            "version": 2,
+            "settings":
+            {
+                "TurnOn":
+                {
+                    "label": "Enable",
+                    "description": "When enabled, It will write Time Left HHMMSS on the display",
+                    "type": "bool",
+                    "default_value": true
+                }
+            }
+        }"""
+
+    def execute(self, data):
+        if self.getSettingValueByKey("TurnOn"):
```

Comment:
bit of nitpicking;

You can make this a bit easier to read by using the inverse. Eg; 
```python3
if not self.getSettingValueByKey("TurnOn"):
    return data
```
That way you don't have to ident the entire block that's below it.